### ISSUE TITLE
Getting Regenerator Run Time issue when using this Lib, I already have Babel configured for aysnc wait

### ISSUE THREAD

**Issue by rahulrsingh09** [adobe/redux-saga-promise#4]
```
/Users/rahsinwb/Documents/Mine/code/SSR/node_modules/@adobe/redux-saga-promise/dist/index.js:27
regeneratorRuntime.mark(implementPromiseAction),
^

```

**Comment by ronen** [comment]
There's not enough info above to know what problem you're facing.   But in any case I can't imagine why this library would be any different from any others you're using that use `regeneratorRuntime`.   

 If you look at `@adobe/redux-saga-promise/dist/index.js` you can see it simply presumes that `regeneratorRuntime` is available at global scope.  If your babel setup isn't making that available, you may need to manually require or import `"regenerator-runtime/runtime"` before anything else, as per [regenerator-runtime README](https://github.com/facebook/regenerator/tree/master/packages/regenerator-runtime#regenerator-runtime)

**Comment by rahulrsingh09** [comment]
Currently we have confugured web pack to do that.. we are using @babel/plugin-transform-runtime for this.

**Comment by ronen** [comment]
Ah, I see.  `@babel/plugin-transform-runtime` doesn't create a global `regeneratorRuntime`, it explicitly imports it for each use.   That's actually a better way to do things.

I've just released v1.1.1 which also builds using `@babel/plugin-transform-runtime`, so doesn't depend on a global `regeneratorRuntime`.  That should fix things for you.
